apps/storybook/package.json

```diff
@@ -2,30 +2,33 @@
   "name": "nameguard-storybook",
   "private": true,
   "scripts": {
+    "build": "npm run tailwind && storybook build",
     "dev": "storybook dev -p 6006",
-    "build": "storybook build"
+    "tailwind": "tailwindcss -i ./styles/input.css -o ./styles/output.css"
```

Comment:
Can we please document in the README.md file what this command is for and the goal it is intended to achieve?